mappings/net/minecraft/world/ModifiableWorld.mapping

```diff
@@ -29,6 +29,11 @@ CLASS net/minecraft/class_1945 net/minecraft/world/ModifiableWorld
 			COMMENT the bitwise flag combination, as described above
 		ARG 4 maxUpdateDepth
 			COMMENT the limit for the cascading block updates
+	METHOD method_30093 breakBlock (Lnet/minecraft/class_2338;ZLnet/minecraft/class_1297;I)Z
+		ARG 1 pos
+		ARG 2 drop
+		ARG 3 breakingEntity
+		ARG 4 maxUpdateDepth
```

Comment:
What does this parameter do?

Reply:
The last one? I copied the name from setBlockState, because that's where it ends up.
That method says this: * @param maxUpdateDepth the limit for the cascading block updates

Reply:
Oh, also: where do the javadocs come from??? Is this also done via mappings?